src/backend/docker-compose.yml

```diff
@@ -118,5 +117,4 @@ volumes:
 
 # Define the absolute path for PROJECT_ROOT
 x-environment:
-  PROJECT_ROOT: ${PROJECT_ROOT}
-
```

Comment:
Let's not make these changes where it is not required. It will create a lot of merge request conflicts in the future

Reply:
ok sir